app/helpers/StringHelper.php

```diff
@@ -392,3 +412,86 @@ function urilize($path, bool $noTime = false): string
 function implodeCsv($value) {
     return implode(', ', $value);
 }
+
+/**
+ * Returns true if the value is an IPv6 or IPv4 address (in that order)
+ */
+function isIpAddress($value) {
```

Comment:
Can you please put a carriage return to all the `function() {` in that file ? :3

Reply:
Did that for everything I touched. Fixing that style for your own code should go outside this PR ;).